### ISSUE TITLE
typescript-multiple-files/index.handlebars does not exists

### ISSUE THREAD

**Issue by Izhaki** [dotansimha/graphql-code-generator#100]
When running (from an npm script)

    gql-gen --url http://0.0.0.0:8889/graphql --template ts-multiple --out ./outM

one gets:

```
Error: Template file /Users/roey/generators/typescript-multiple-files/index.handlebars does not exists!
    at Object.exports.loadFromPath (/Users/roey/Development/gql2ts/node_modules/graphql-code-generator/dist/loaders/template-loader.js:11:15)
    at Object.exports.compileTemplate (/Users/roey/Development/gql2ts/node_modules/graphql-code-generator/dist/loaders/template-loader.js:30:39)
    at Transform (/Users/roey/Development/gql2ts/node_modules/graphql-code-generator/dist/engine/transform-engine.js:85:44)
    at process._tickCallback (internal/process/next_tick.js:103:7)
```

Notice that for some reason it searches for the file in the home folder (`/Users/roey/generators/...`).

Full `package.json`:

```
{
  "name": "gql2ts",
  "version": "1.0.0",
  "description": "",
  "main": "index.js",
  "scripts": {
    "test": "echo \"Error: no test specified\" && exit 1",
    "emit": "gql-gen --url http://0.0.0.0:8889/graphql --template typescript --out ./out/typings.d.ts",
    "em": "gql-gen --url http://0.0.0.0:8889/graphql --template ts-multiple --out ./outM"    
  },
  "author": "",
  "license": "ISC",
  "devDependencies": {
    "graphql-code-generator": "^0.5.2"
  },
  "dependencies": {
    "graphql": "^0.9.6"
  }
}
```
Note that `emit` works, but `em` doesn't.

**Comment by Izhaki** [comment]
I don't think [this](https://github.com/dotansimha/graphql-code-generator/blob/9ccde1d96d09e43a7d75c44889a65cf3653937d9/src/engine/templates.ts#L41) resolves correctly. And this is probably the fix:

https://stackoverflow.com/questions/12238477/determine-command-line-working-directory-when-running-node-bin-script

**Comment by tafkanator** [comment]
probably solves once this issue is solved #49

**Comment by zephraph** [comment]
This will be fixed with #110

**Comment by zephraph** [comment]
This is fixed in master, it's just awaiting a release.

**Comment by dotansimha** [comment]
Fixed in 0.5.5 :)
